frontend/src/Frontend/Page/Resources.hs

```diff
@@ -9,12 +9,13 @@
 
 module Frontend.Page.Resources (resources) where
 
+import Obelisk.Frontend.GoogleAnalytics
 import Data.Foldable (for_)
 import Data.Text (Text)
 import Reflex.Dom
 import Frontend.CommonWidgets
 
-resources :: DomBuilder t m => Section m
+resources :: (Analytics t GtagJSCall m, DomBuilder t m) => Section m
```

Comment:
`GtagJSCall`... What's this telling me?

Reply:
`GtagJSCall` is a raw call to Google's `gtag.js` client library;  where the function simply adds the event to the end of a javascript array in a global variable.

The intention is that applications could define their own events that they want to tell some analytics platform about,  then define a mapping from those events onto the raw calls.   But in this case, we are just using the raw calls directly.

Reply:
But isn't this from `Obelisk.Frontend.GoogleAnalytics`? Why should I be interested in supporting other analytics platforms?